### PR TITLE
main content

### PR DESCRIPTION
**Pull request by coder-mezab**
I have created the main content of the web site let me know what you guys think

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -65,17 +84,111 @@ nav ul li a {
...
+  color: whitesmoke;
+  background: -webkit-linear-gradient(top, #b20094 0%, #ff44f2 100%);
   border-radius: 20px;
   transition: background 0.5s;
 }
 .btn:hover {
   background: #9b1a1a;
   box-shadow: 2px 2px 4px 2px black;
 }
+.categeries {
```
Comment (faran4hatch) [inline css/style.css:101]: it should be `categories` correcting typo

File: index.html [hunk 2]
```diff
@@ -12,21 +12,22 @@
...
     />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   </head>
   <body>
     <div class="container">
       <div class="navbar">
         <div><img class="logo" src="css/images/logo.png" /></div>
         <nav>
-          <ul>
+          <ul id="MenuItems">
```
Comment (faran4hatch) [inline index.html:22]: lowercase class names please

Comment (faran4hatch) [reply index.html:22]: seperated by hyphens -> `menu-items`

File: index.html [hunk 3]
```diff
@@ -45,5 +46,229 @@ <h1>
         </div>
       </div>
     </div>
+    
+    <div class="categeries">
+      <div class="haha"> <h1> category-1 </h1> </div>
```
Comment (faran4hatch) [inline index.html:51]: what is this haha class... haha 😆

File: index.html [hunk 4]
```diff
@@ -45,5 +46,229 @@ <h1>
...
+              <p>Make Your Life Smart With The New<br>
+              Much Powerful Less Consuming SmartWatch <strong style="color: red;">ColorOS3.0</strong>  </p>
+              <a href="" class="btn"> Explore Now &#8594; </a>
+            </div>
+            <div class=c1>
+              <img src="css/images/watch.jpg">
+              </div>
+          </div>
+          <div class="rew">
+          <div class="t1">
```
Comment (faran4hatch) [inline index.html:227]: whats t1 ? give logical names that we can relate to !!

File: index.html [hunk 5]
```diff
@@ -45,5 +46,229 @@ <h1>
...
+                <i class="fa fa-star" ></i>
+                <i class="fa fa-star-o" ></i>
+              </div>
+              <h4>449.99$</h4>         
+             
+          </div>
+          </div>
+          <div class="banner">
+            
+            <div class="c1">
```
Comment (faran4hatch) [inline index.html:213]: whats c1 🤨

File: css/style.css [hunk 6]
```diff
@@ -45,9 +66,7 @@ nav ul li a {
   align-items: center;
   flex-wrap: wrap;
 
-  margin: auto;
   background: -webkit-radial-gradient(#fff, #c9c506);
-  position: relative;
 }
 .col-1 {
```
Comment (Tauqeer-Ahmed-99) [inline css/style.css:71]: Which part of html content this selector targets?
Be careful while targeting the elements you have used `c1`
in html file and here targeting `c-1`
@mezab

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]
Other than using class names making no sense to the content, code looks good to me...
Approving..

File: css/style.css (hunk 1 above)
Comment (coder-mezab) [reply css/style.css:101]: ok

File: index.html (hunk 2 above)
Comment (coder-mezab) [reply index.html:22]: okay okay

File: index.html (hunk 3 above)
Comment (coder-mezab) [reply index.html:51]: i will change it

**Review by fayez-baig — APPROVED** [review]
👍🏼

**Pull request merged**